### DISCUSSION TITLE
Variable in prisma.$table.updateMany, it is possible?

### DISCUSSION THREAD

**Discussion by robokonk** [prisma/prisma#12884] [Q&A] [answered]
I want building several miniFunction for prisma like this:
```


 const prismaUpdateMany = async (table, whereVariable, dataVariable) => {
 return await prisma.table.updateMany({
   where: {
     whereVariable,
   },
   data: {
     dataVariable,
   },
 });
 }
```

How can I put variable in this place (table)?
```
 prisma.$table.updateMany
```

Maybe my idea is incorrect, what I should do in this case, to achieve a similar goal?

**Comment by robokonk** [comment]
Bellow my real code, I need linking tableName with prisma.tableName.
![image](https://user-images.githubusercontent.com/103321734/164214590-e9598e9d-fba8-4a2e-b09c-09c5fd31ddeb.png)

**Comment by psikoi** [comment, marked as answer]
`await prisma[tableName].findMany(...)` might be your best bet

you'd want to use the value of the `tableName` variable as the object key, not the variable name

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
